.babelrc

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
To keep your repo more organized, files like the linters config files should be ignored inside .gitignore

Reply:
Done